models/solutions.js

```diff
@@ -27,7 +27,10 @@ module.exports = {
       programDescription: String,
       entityProfileFieldsPerEntityTypes: Object,
       startDate: Date,
-      endDate: Date,
+      endDate: {
```

Comment:
@priyanka-TL why this field is an index and also if this is required then why not startDate?

Reply:
we are searching with end date only. we are not checking the start date.